src/thunderbots/software/util/ros_messages.h

```diff
@@ -96,5 +96,45 @@ namespace Util
             const thunderbots_msgs::RefboxCommand& command);
 
         World createWorldFromROSMessage(const thunderbots_msgs::World& world_msg);
+
+        /**
+         * Given a World message, it will transform the world's orientation
+         * Used when changing defending side
+         *
+         * @param old_world_msg The world message to transform
+         * @return The transformed world message
+         */
+        thunderbots_msgs::World transformWorldMessage(
+            const thunderbots_msgs::World& old_world_msg);
+
+        /**
+         * Given a Ball message, it will transform the ball's orientation
+         * Used when changing defending side
+         *
+         * @param old_ball_msg The ball message to transform
+         * @return The transformed ball message
+         */
+        thunderbots_msgs::Ball transformBallMessage(
```

Comment:
As a suggestion, we could rename all these functions to `invertMsgFieldSide` (so have `invertMsgFieldSide(Ball b)`, `invertMsgFieldSide(World w)`, etc.)?